develop/components/pages/AccordionPage.tsx

```diff
@@ -1,15 +1,12 @@
 import React from "react";
-import { Accordion, AccrodionListItem } from "../../../src/Accordion/Accordion";
+import { Accordion } from "../../../src/Accordion/Accordion";
 import Highlight from "react-highlight";
 import { loremIpsum } from "lorem-ipsum";
+import { AccordionItemProps, AccordionItem } from "../../../src/Accordion/AccordionItem";
 const docMD: string = require("../../../src/Accordion/readme.md");
 
 const AccordionPage: React.FC = () => {
-    const [lists] = React.useState<Array<Array<AccrodionListItem>>>(
-        Array(4)
-            .fill(0)
-            .map(() => generateList())
-    );
+    const [lists] = React.useState<Array<Array<AccordionItemProps>>>([...Array(4)].map(() => generateList()));
```

Comment:
```
    const lists: Array<Array<AccordionItemProps>>> = React.useMemo(() =>[...Array(4)].map(() => generateList()), []);
```
You are not setting it anywhere or are you ? Use memo instead.

Reply:
Also I'm really not a fan of `Array<Array<T>>` it just looks so ugly. `T[][]`looks much better

Reply:
@DanSallau 
I only generate once and it's stored in a state, I don't think I need to use memo especially since it's meant to generate different results every time. What's the point of memoizing the results. This is only meant to make it easier for us developers to add more examples by just increasing the number which is currently **4 = 4 examples**

@mario-subo 
We have a standard of using this which is `Array<T>` and it's everywhere. If it comes to preference, I prefer this one because for me it's not about having a short version of it. It's about readability. Using `T[][]` might confuse the reader about more complicated structures such as:
```typescript
type A = Array<[Array<[T]>, K]>; // Current standard
type B = [[T], K][]; // Suggested standard
```

Reply:
I think your explanation above is a textbox definition why you need the useMemo. You can memoized it without changing the state. I still believe useMemo is more appropriate here since you don't update your state.

Reply:
```typescript
  type A = Array<Array<T>, K>;
```

Reply:
```typescript
let example: A = [
    [
        [1, 2, 3],
        1
    ],
    [
        [1, 2, 3],
        1
    ]
]
```

Reply:
```typescript
interface ReactChildren {
    map<T, C>(children: C | C[], fn: (child: C, index: number) => T): C extends null | undefined ? C : Array<Exclude<T, boolean | null | undefined>>;
    forEach<C>(children: C | C[], fn: (child: C, index: number) => void): void;
    count(children: any): number;
    only<C>(children: C): C extends any[] ? never : C;
    toArray(children: ReactNode | ReactNode[]): Array<Exclude<ReactNode, boolean | null | undefined>>;
}
```